src/display/editor/outliner.js

```diff
@@ -528,12 +534,20 @@ class FreeOutliner {
     const lastTop = this.#last.subarray(4, 6);
     const lastBottom = this.#last.subarray(16, 18);
     const [x, y, width, height] = this.#box;
+    const lastTopX =
+      (this.#lastX + (lastTop[0] - lastBottom[0]) / 2 - x) / width;
+    const lastTopY =
+      (this.#lastY + (lastTop[1] - lastBottom[1]) / 2 - y) / height;
+    const lastBottomX =
+      (this.#lastX + (lastBottom[0] - lastTop[0]) / 2 - x) / width;
+    const lastBottomY =
+      (this.#lastY + (lastBottom[1] - lastTop[1]) / 2 - y) / height;
```

Comment:
Would it make sense to add a helper method for this, given that same code seems to be repeated below as well?
E.g. something like
```javascript
  #getLastCoords() {
    const lastTop = this.#last.subarray(4, 6);
    const lastBottom = this.#last.subarray(16, 18);
    const [x, y, width, height] = this.#box;

    return [
      (this.#lastX + (lastTop[0] - lastBottom[0]) / 2 - x) / width,
      (this.#lastY + (lastTop[1] - lastBottom[1]) / 2 - y) / height,
      (this.#lastX + (lastBottom[0] - lastTop[0]) / 2 - x) / width,
      (this.#lastY + (lastBottom[1] - lastTop[1]) / 2 - y) / height,
    ];
  };
```
which would change this code to 
```suggestion
    const [lastTopX, lastTopY, lastBottomX, lastBottomY] = this.#getLastCoords();
```